arbnode/transaction_streamer.go

```diff
@@ -1473,8 +1473,9 @@ func (s *TransactionStreamer) checkSubmittedTransactionForFinality(ctx context.C
 		resp.VidCommon,
 	)
 
-	if !namespaceOk {
-		return fmt.Errorf("error validating namespace proof (height: %d)", height)
+	if err != nil || !namespaceOk {
```

Comment:
Do we need to check both that `err != nil` and `bool == true`. Just a nit but both seem to happen together

Reply:
So the error only needs to be thrown when err is nil or namespace is not ok

Reply:
I think Jean is suggesting that we should return only one error.

If error is nil, then verification is ok.

We can do it in subsequent PRs

Reply:
Yes exactly Jeremy, that is what I meant!